Improve security hub docs (#7649)
wazuh/wazuh-documentation: source/cloud-security/amazon/services/supported-services/security-hub.rst

   #. Enabling Amazon Security Hub.
   #. Integrating Security Hub with EventBridge.

      EventBridge allows storing Security Hub findings and insights in S3 buckets.

>       There are three types of events available as follows. Each type of event uses a specific `EventBridge event format <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-event-formats.html>`__. The Wazuh integration takes from the events every relevant ``detail`` field and value along with the ``detail-type`` value.

Reviewer: ```suggestion
      There are three types of events available, each type uses a specific `EventBridge event format <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-event-formats.html>`__. The Wazuh integration takes every relevant ``detail`` and ``detail-type`` values from them.
```

   .. code-block:: json

      {
         "Version": "2012-10-17",
         "Statement": [
            {
                  "Sid": "SecurityHubAllowAll",
                  "Effect": "Allow",
                  "Action": "securityhub:*",
                  "Resource": "*"
            },
            {
                  "Sid": "SecurityHubServiceLinkedRole",
                  "Effect": "Allow",
                  "Action": "iam:CreateServiceLinkedRole",
                  "Resource": "*",
                  "Condition": {
                     "StringLike": {
                        "iam:AWSServiceName": "securityhub.amazonaws.com"
                     }
                  }
            },
            {
                  "Sid": "OtherServicePermission",
                  "Effect": "Allow",
                  "Action": [
                     "guardduty:GetDetector",
                     "guardduty:ListDetectors",
                     "inspector2:BatchGetAccountStatus",
                     "pricing:GetProducts"
                  ],
                  "Resource": "*"

>             }

Reviewer: Indentation here has 5 spaces instead of 3

                     "organizations:DescribeAccount",
                     "organizations:DescribeOrganizationalUnit"
                  ],
                  "Resource": "*"
            },
            {
                  "Sid": "OrganizationPermissionsEnable",
                  "Effect": "Allow",
                  "Action": "organizations:EnableAWSServiceAccess",
                  "Resource": "*",
                  "Condition": {
                     "StringEquals": {
                        "organizations:ServicePrincipal": "securityhub.amazonaws.com"
                     }
                  }
            },
            {
                  "Sid": "OrganizationPermissionsDelegatedAdmin",
                  "Effect": "Allow",
                  "Action": [
                     "organizations:RegisterDelegatedAdministrator",
                     "organizations:DeregisterDelegatedAdministrator"
                  ],
                  "Resource": "arn:aws:organizations::*:account/o-*/*",
                  "Condition": {
                  "StringEquals": {
                     "organizations:ServicePrincipal": "securityhub.amazonaws.com"
                     }

>                   }

Reviewer: Indentation here has 5 spaces instead of 3

Integrating Security Hub with EventBridge
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> To integrate Security Hub with EventBridge, you need to create the following:

Reviewer: I would add `resources` at the end of this sentence

```
To integrate Security Hub with EventBridge, you need to create the following resources:
```
